CHANGELOG.md

```diff
@@ -40,6 +40,7 @@
 
 ## Deprecations
```

Comment:
Did we have this section in previous releases? If not, we need an emoji, to be in line with the other sections. How about one of those?

🗑️
🚩
👎

Reply:
We didn't. I like the flag one. Added.